timemachine/cpp/src/bd_exchange_move.cu

```diff
@@ -10,11 +10,13 @@
 #include "math_utils.cuh"
 #include "mol_utils.hpp"
 
+namespace timemachine {
 // The number of threads per block for the setting of the final weight of the moved mol is low
 // if using the same number as in the rest of the kernels of DEFAULT_THREADS_PER_BLOCK
-#define WEIGHT_THREADS_PER_BLOCK 512
-
-namespace timemachine {
+static const int WEIGHT_THREADS_PER_BLOCK = 512;
+// The number of translations to generate each step. The first three values are a unit vector translation and the fourth
+// value is used for the metropolis hasting check
+static const int BD_TRANSLATIONS_PER_STEP_XYZW = 4;
```

Comment:
Not sure if this has become an established convention, but I'd prefer something other than `W` for the noise (acceptance probability?) since we use `(x, y, z, w)` tuples elsewhere to represent 4-d coordinates.

Reply:
Seems a bit confusing to call this "translations per step" when (IIUC) the x, y, and z components describe a translation and the fourth component is a uniform sample? Would something like "random samples per step" be appropriate?

Reply:
> Not sure if this has become an established convention

Don't think this is a convention, no preference from me. 

> Would something like "random samples per step" be appropriate?

Think that would be more confusing since there are actual random samples and a good number of types of randomness.